### DISCUSSION TITLE
Possible to add *just one more comment* to an old bug around `prefer-destructuring` ?

### DISCUSSION THREAD

**Discussion by nexushoratio** [eslint/eslint#17521] [General] [answered]
I couldn't find any guidance on this.  Pointers to existing *old bugs policy* appreciated.

While looking at something, I came across issue #11836 which was filed back in 2019.

I'm new to JS, so I'm not sure if there was no way to do the recommended destructuring at the time.  It can be done now if ugly:
```
let obj1, obj2;

({property2: obj1.property1} = obj2);
```

Would it make sense for a team member to update that and officially document it as `not-a-bug` (for future folks who come across it).  Or would such a policy just cause too many problems with folks always wanting to do that?

**Comment by mdjermanovic** [comment, marked as answer]
Makes sense to me, thanks for the suggestion! I've updated the issue now.
